SoftLayer/CLI/vlan/list.py

```diff
@@ -63,3 +75,16 @@ def cli(env, sortby, datacenter, number, name, limit):
         ])
 
     env.fout(table)
+
+
+def get_pod_with_closed_announcement(vlan, pods):
+    """Gets pods with announcement to close"""
+    for pod in pods:
+        if utils.lookup(pod, 'backendRouterId') == utils.lookup(vlan, 'primaryRouter', 'id') \
+                or utils.lookup(pod, 'frontendRouterId') == utils.lookup(vlan, 'primaryRouter', 'id'):
+            if 'CLOSURE_ANNOUNCED' in utils.lookup(pod, 'capabilities'):
+                name_pod = utils.lookup(pod, 'name').split('.')[1] + '*'
+                return name_pod.capitalize()
```

Comment:
I like the idea of adding color for these. This will only highlight the word "pod01" or whatever, but I think thats ok for now, as the formatting.Table class doesn't support anything fancier for now.
```suggestion
                return "[red]" + name_pod.capitalize()   + "[/red]"
```

![image](https://user-images.githubusercontent.com/7408017/181846898-f5984119-4da5-4dbd-aa4a-574708107c84.png)

Reply:
Done